plugins/woocommerce/includes/wc-core-functions.php

```diff
@@ -1576,8 +1576,10 @@ function wc_back_link( $label, $url ) {
 function wc_help_tip( $tip, $allow_html = false ) {
 	if ( $allow_html ) {
 		$sanitized_tip = wc_sanitize_tooltip( $tip );
+		$aria_label    = wc_sanitize_tooltip( wp_strip_all_tags( $tip ) );
```

Comment:
```suggestion
		$aria_label    = wp_strip_all_tags( $tip );
```

It looks like [wc_sanitize_tooltip](https://github.com/woocommerce/woocommerce/blob/e40d91c2799143102a60dddd9d9e598bdada8f1e/plugins/woocommerce/includes/wc-formatting-functions.php#L427-L451) just does two things: `wp_kses` with a special set of allowed HTML tags, and then `htmlspecialchars`. Since we want to just strip all tags here, we can probably omit `wc_sanitize_tooltip` altogether...

Reply:
I guess this combined with my other comment means we could potentially move the `$aria_label` assignment out of the conditional altogether, since it would be the same regardless of the value of `$allow_html`...